backend/app/views/spree/admin/shared/_content_header.html.erb

```diff
@@ -1,23 +1,13 @@
 <% if content_for?(:page_title) || content_for?(:page_actions) %>
   <div id="content-header" data-hook>
-    <div class="container">
-      <div class="sixteen columns">
-        <div class="block-table">
-          <% if content_for?(:page_title) %>
-            <div class="table-cell">
-              <h1 class="page-title <%= yield :page_title_classes %>"><%= yield :page_title %></h1>
-            </div>
-          <% end %>
+    <h1 class="page-title <%= yield :page_title_classes %>"><%= yield :page_title %></h1>
```

Comment:
This doesn't work quite like this since `.page-actions` is applied to the `ul` which is inside of that `div` with the `toolbar` class and datahook.  Works when applied to the toolbar div obviously but are we keeping that?  I would like to eliminate the empty h1 even though it shouldn't hurt, empty elements used to influence aren't great.